### ISSUE TITLE
JSON input validation error - infiltration height

### ISSUE THREAD

**Issue by BrianaParrBaker** [fishbaugher/WAweb-Issues#85]
Error message when attempting to run audit:
Audit info: 
![GG TEST](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/4cbacdee-ecb0-4e5a-a6f0-72c76391425a)

Error message:
![Goldschmidt error](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/800e52ed-3a0d-4148-b02b-5a7486027e9c)

![detail](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/0bb03494-9b03-4090-9746-8168be42691d)

**Comment by BrianaParrBaker** [comment]
Note: I originally ran this audit a couple weeks ago without issue. I attempted to rerun it today and was received the error message about the infiltration height.

**Comment by fishbaugher** [comment]
Briana, I just tried to run 1102 again with the 18' infiltration height, and it ran without error.

It make me wonder about how ORNL is doing releases.  I know the analysis engine and the UI can be release separately, and for now the UI still reports a version number of 10.05.015 but there is a way to peer into the analysis engine version which reports:

v10.05.015 Thu Feb 29 13:36:39 EST 2024

I am going to close this issue since it would appear the JSON range check error has been fixed, just not sure when or in what version.
